sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -555,9 +562,9 @@ public void testReadingDisplayData() {
         hasLabel("Table ID"),
         hasValue("fooTable"))));
 
-    assertThat(displayData, hasDisplayItem("rowFilter", rowFilter.toString()));
+    assertThat(displayData, hasDisplayItem("rowFilter", ValueProvider.StaticValueProvider.of(rowFilter).toString()));
```

Comment:
I think should work, although it may be worth expanding the `ValueProvider.StaticValueProvider.of(rowFilter).toString()` to make sure it doesn't pick up the default (`StaticVlaueProvider#8782350;` won't be super useful). It may be tested elsewhere, but we should at least confirm.